src/components/UserInfo/UserInfo.jsx

```diff
@@ -1 +1,12 @@
-export const UserInfo = () => <>Put the user here</>;
+import React from 'react';
+
+export const UserInfo = ({ user }) => {
+  return (
+    <div className="UserInfo">
+      <p className="UserInfo__name">{user.name}</p>
+      <a className="UserInfo__email" href={`malito:${user.email}`}>
```

Comment:
There is a typo in the href attribute value. It should be 'mailto:' instead of 'malito:'.